### PR TITLE
Generate welsh schema for CRIS

### PR DESCRIPTION
**Pull request by Hajara-I**
### What is the context of this PR?

- Added `.po` file to repo
- Generated Welsh schema for CRIS.

### How to review
Run the command `make test-translation-templates` to check that the schemas have been translated successfully and are up-to-date.

### CODE REVIEW

**Comment by Hajara-I** [comment]
[Generate/Update Welsh Schema for PHM/CRIS (1 SP)](https://trello.com/c/p2QNOV1A/5623-generate-update-welsh-schema-for-phm-cris-1-sp)

**Review by MebinAbraham — APPROVED** [review]

**Comment by MebinAbraham** [comment]
Until the whitespace in the PO file is fixed, the schemas will keep erroring.

File: schemas/health/cy/phm_0001.json [hunk 1]
```diff
@@ -838,7 +838,7 @@
                                 "id": "question23530584-48bc-45c5-bb02-e662a5516b50",
```
Comment (MebinAbraham) [inline schemas/health/cy/phm_0001.json:838]: Would this have errored if you didn't regenerate the CY schema?

Comment (Hajara-I) [reply schemas/health/cy/phm_0001.json:838]: Hi Mebin can you expand on this comment some more? 

I generated the `cris_0001.json` schema by running the `make translate-schemas` command in the README file after adding the `.po` file into the repo.

**Review by Yuyuutsu — APPROVED** [review]
Tested and schema passed in a local validator run

**Comment by Yuyuutsu** [comment]
Retested 👍 looks good to me

**Pull request merged**